src/sentry/ownership/grammar.py

```diff
@@ -18,10 +19,15 @@
 from sentry.utils.codeowners import codeowners_match
 from sentry.utils.event_frames import find_stack_frames, get_sdk_name, munged_filename_and_frames
 from sentry.utils.glob import glob_match
-from sentry.utils.safe import PathSearchable, get_path
+from sentry.utils.safe import get_path
 
 __all__ = ("parse_rules", "dump_schema", "load_schema")
 
+if TYPE_CHECKING:
+    NodeVisitor = BaseNodeVisitor[str]
+else:
+    NodeVisitor = BaseNodeVisitor
```

Comment:
what's this needed for?

Reply:
There might be an easier way(?) but we only have the generic version for type checking (https://github.com/getsentry/sentry/blob/4aa527a59d36d4dc20741564d88912c4bd1b0a5e/requirements-dev.txt#L48) so without this python complains that it's not subscriptable

Reply:
it's generic at typing time but not at runtime